### ISSUE TITLE
Will data outside the Gulf of Mexico be available eventually?

### ISSUE THREAD

**Issue by ddbuscombe** [gantian127/bmi_dbseabed#3]
If so, an approximate timeline would be great - thanks a lot

**Comment by gantian127** [comment]
@dbuscombe-usgs Yes, the datasets will eventually cover a larger area (hopefully global scale in the future). I will ask my collaborator who created these datasets. He will be able to answer your question. Thank you!

**Comment by ddbuscombe** [comment]
Thanks, give my best wishes to Chris
